### DISCUSSION TITLE
Can someone help me figure out how this repository works as a package?

### DISCUSSION THREAD

**Discussion by shreyastaware** [py-pdf/pypdf#834] [Q&A]
I am thinking to build my own pdf conversion library for specific features that can implemented on images using opencv but not using this library. Any idea/ brief overview on how this repo works internally? It would be great if you could give me a starting point for creating a library based off of this.

**Comment by MartinThoma** [comment]
This question is way to broad for me to answer. Can you be more specific? What did you understand?
